toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -179,6 +179,26 @@ SystemConfig:
       Permissions: "664"
 ```
 
+## KernelCommandLine type
+
+Options for configuring the kernel.
+
+### ExtraCommandLine
+
+If the partitions are customized, then the `grub.cfg` file will be reset to handle the
+new partition layout.
+When the new `grub.cfg` file is created, the `ExtraCommandLine` value will be added to
+the Linux kernel command line options.
+
+This field may only be used when when the [Disks](#disks-disk) and
+[SystemConfig.PartitionSettings](#partitionsettings-partitionsetting) fields are
+specified.
+
+### ExtraCommandLineAdd
+
+Additional Linux kernel command line options that are added to the existing `grub.cfg`
```

Comment:
I dont see a reason, from user perspective, to have these two varied attributes. Why should user care and differentiate between the ExtraCommandLine and ExtraCommandLineAdd?

Reply:
Fair enough. I merged them.